src/en/wizden-staff/maintainer/adding-new-maintainers.md

```diff
@@ -0,0 +1,36 @@
+# Adding New Maintainers
+
+This document outlines the process for adding new maintainers to Wizden, it is intended for the current maintainers and project managers to follow when adding new maintainers to the project. 
+
+## Requirements
+
+As maintainers represent wizden as staff, certain requirements must be met before a person can be a maintainer. These requirements are as follows:
+- Be a member of the Space Station 14 Discord server, as many discussions happen there.
+- Not have had any singificant administrative issues in the past, such as being banned from the game or Discord server or having a history of being rude to other members of the community.
+    - This is a subjective requirement, and should be discussed in the voting process.
+- Have a good understanding of the codebase and how it works.
+    - This is a subjective requirement, and should be discussed in the voting process.
+
+### Voting Process
+
+Before a vote can start, the person being voted on must be nominated by a current maintainer. Once a person has been nominated, the following process should be followed:
```

Comment:
This reads a bit strange and confusing to me, it kinda implies the existence of a state where the candidate is being considered, but vote can't be started until they get nominated... Which is actualy how their name comes up in the first place, of course.
This seems more straightforward to me:

```suggestion
To add a new maintainer, a current maintainer has to nominate the candidate. Once a person has been nominated, the following process should be followed:
```